react-form-controls/my-app/src/RegistrationFormControlled.js

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+
+export default function RegistrationFormControlled() {
+  const [user, setUser] = useState('');
+  const [pass, setPass] = useState('');
+  function handleSubmit(e) {
+    e.preventDefault();
+    console.log('Controlled', { user, pass })
+  }
+
+  return (
+    <form onSubmit={(e) => handleSubmit(e)}>
```

Comment:
Since handleSubmit takes 'e' as a parameter, you could just pass `handleSubmit` directly to `onSubmit`. No need to use an extra arrow function.